xchange-bitstamp/src/main/java/info/bitrich/xchangestream/bitstamp/v2/BitstampStreamingMarketDataService.java

```diff
@@ -0,0 +1,77 @@
+package info.bitrich.xchangestream.bitstamp.v2;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import info.bitrich.xchangestream.bitstamp.dto.BitstampOrderBook;
+import info.bitrich.xchangestream.bitstamp.dto.BitstampWebSocketTransaction;
+import info.bitrich.xchangestream.core.StreamingMarketDataService;
+import info.bitrich.xchangestream.service.netty.StreamingObjectMapperHelper;
+import io.reactivex.Observable;
+import org.knowm.xchange.bitstamp.BitstampAdapters;
+import org.knowm.xchange.currency.CurrencyPair;
+import org.knowm.xchange.dto.marketdata.OrderBook;
+import org.knowm.xchange.dto.marketdata.Ticker;
+import org.knowm.xchange.dto.marketdata.Trade;
+import org.knowm.xchange.exceptions.NotAvailableFromExchangeException;
+
+import java.util.Date;
+
+/**
+ * Bitstamp WebSocket V2 Streaming Market Data Service implementation
+ * Created by Pavel Chertalev on 15.03.2018.
+ */
+public class BitstampStreamingMarketDataService implements StreamingMarketDataService {
+    private final BitstampStreamingService service;
+
+    public BitstampStreamingMarketDataService(BitstampStreamingService service) {
+        this.service = service;
+    }
+
+    public Observable<OrderBook> getDifferentialOrderBook(CurrencyPair currencyPair, Object... args) {
```

Comment:
Minor formatting

Reply:
Not clear, what formatting looks bad?

Reply:
Oh it might just be GitHub, in which case ignore me.